composer.json

```diff
@@ -13,7 +14,9 @@
 	},
 	"require": {
 		"php": "^8.1",
-		"dnj/laravel-account": "dev-master"
+		"dnj/laravel-account": "dev-master",
+		"dnj/laravel-currency": "^1.0.1",
+		"laravel/legacy-factories": "^1.3"
```

Comment:
why did you need legacy-factories?

Reply:
This was for the previous changes I made to the User Factory
done